data_structures/avltree_map.cpp

```diff
@@ -0,0 +1,226 @@
+/**
+ * @file avltree_map.cpp
+ * @brief Unit testings for avltree_map.hpp
+ * @author [r.ivance](https://github.com/RIvance)
+ * @see avltree_map.hpp
+ */
+
+#include "avltree_map.hpp"
+
+#include <map>
+#include <iostream>
+#include <set>
+#include <algorithm>
```

Comment:
Please add a one-line description of what the library/header is for (**see the example below**).

```c++
#include <iostream>    /// for IO operations
#include <cassert>    /// for assert
```